server/db/models/event.js

```diff
@@ -0,0 +1,56 @@
+const { Model, STRING, INTEGER, UUID, UUIDV4, BOOLEAN, TEXT, DATE, TIME } = require('sequelize');
+const db = require('../connection');
+
+class Event extends Model{}
+
+Event.init(
+    {
+        id: {
+            type: UUID,
+            defaultValue: UUIDV4,
+            unique: true,
+            primaryKey: true,
+        },
+        name: {
+            type: STRING,
+            allowNull: false,
+            validate: {
+            notEmpty: true
+            }
+        },
+        description: {
+            type: TEXT,
+            allowNull: true
```

Comment:
I think we should require descriptions for events.

Reply:
Sounds good, I'll change it.